js/buttons.js

```diff
@@ -0,0 +1,50 @@
+function getButtons() {
+  const simpleButtons = Array.from(document.getElementsByClassName('simple-button'))
+  const aboutUsButtons = Array.from(document.getElementsByClassName('about-us__simple-button'))
+  const planButtons = Array.from(document.getElementsByClassName('plan__button'))
+  const getStartedButton = Array.from(document.getElementsByClassName('get-started__button'))
+  const slideArrowButton = Array.from(document.getElementsByClassName('slide__arrow'))
+
+  simpleButtons.forEach((elem) => {
+    elem.addEventListener('click', (event) => {
+      event.preventDefault()
+      elem.classList.toggle('simple-button_active')
+    })
+  })
+
+  aboutUsButtons.forEach((elem) => {
+    elem.addEventListener('click', (event) => {
+      event.preventDefault()
+      aboutUsButtons.forEach((item) => {
+        item.classList.remove('about-us__simple-button_active-red')
+      })
+      elem.classList.add('about-us__simple-button_active-red')
+    })
+  })
+
+  planButtons.forEach((elem) => {
```

Comment:
you can create separate function to remove code duplication. You do the same actions almost in every function